src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
+  - Fully exited validators will automatically receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
```

Comment:
```suggestion
  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This Ethereum network upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
```
Maybe worth explicitly mentioning this is a protocol upgrade?

Do we have anything we can link to when we mention the "Shanghai/Capella upgrade"? @jmcook1186?

Reply:
Should we include the word "protocol" here?
```suggestion
  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This Ethereum network protocol upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
```